File: geek-university-algoritmo-e-logica-de-programacao/java/exercicio_29.java
import java.util.Scanner;

public class exercicio_29 {

    public static void main(String[] args) {
        String nome_usuario, senha;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe o nome de usuario: ");
        nome_usuario = teclado.next();
        System.out.print("Informe a senha: ");
        senha = teclado.next();

        while (nome_usuario.equals(senha)) {
            System.out.println("Erro: O nome de usuário e senha devem ser diferentes.");
            System.out.print("Informe o nome de usuario: ");
            nome_usuario = teclado.next();
            System.out.print("Informe a senha: ");
            senha = teclado.next();
        }

        teclado.close();
    }

}

File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_29.c
#include <stdio.h>
#include <string.h>

int main() {
    char nome_usuario[100], senha[100];

    printf("Informe o nome de usuario: ");
    gets(nome_usuario);
    printf("Informe a senha: ");
    gets(senha);

    while (!strcmp(nome_usuario, senha)) {
        printf("Nome de usuario e senha devem ser diferentes.\n");
        printf("Informe o nome de usuario: ");
        gets(nome_usuario);
        printf("Informe a senha: ");
        gets(senha);
    }
}
